README.md

```
@@ -1,2 +1,75 @@
+- [x] .OBJ files
+- [x] Limited cones 
+- [x] Limited cylinders 
+- [ ] Cones 
+- [ ] Cylinders 
+- [ ] Mobius strip 
+- [ ] Fractals
+- [ ] Triangles
+- [ ] Tanglecube
+- [ ] Torus
+
+#### Materials:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)